src/main/java/org/ballerinalang/command/util/ToolUtil.java

```diff
@@ -443,6 +443,9 @@ public static boolean downloadDistribution(PrintStream printStream, String distr
                     conn.setRequestProperty("testMode", "true");
                 }
                 if (conn.getResponseCode() == 302) {
+                    if (!testMode) {
```

Comment:
I am wondering should we commit this code as it will be executed in prod as well?
Ideally this could be coming from a diff/patch file where it has to be applied in order to test locally